ros/angel_msgs/msg/HandJointPose.msg

```diff
@@ -0,0 +1,9 @@
+#
+# Message for transmitting a single hand joint pose.
+#
+
+# Which joint this pose represents
+string joint
```

Comment:
This is probably fine for now, but if we find we have net congestion issues, we can bring this down to the int used in the MRTK enum, and reference that enum here. Or replicate the enum here in ROS2 message style, e.g. https://answers.ros.org/question/9427/enum-in-msg/